project_event/models/project_task.py

```diff
@@ -1178,3 +1178,8 @@ def order_task_fields(tracking_values):
                 if task.__getitem__(2).get('field') == task_fields_list[x]:
                     task_tracking_values.append(task)
         return task_tracking_values
+
+    @api.model
+    def _disable_merge_action(self):
+        self.env.ref('project.task.merge.wizard.project_task_actions_act_window_merge_tasks').unlink()
```

Comment:
check this link : https://www.odoo.com/fr_FR/forum/aide-1/question/use-domain-in-act-window-based-on-uid-132683